x-pack/legacy/plugins/ml/public/components/rule_editor/utils.js

```diff
@@ -15,6 +15,7 @@ import { cloneDeep } from 'lodash';
 import { ml } from '../../services/ml_api_service';
 import { mlJobService } from '../../services/job_service';
 import { i18n } from '@kbn/i18n';
+import { WIZARD_TYPE } from '../../jobs/new_job/simple/components/constants/general';
```

Comment:
Should the values out of `ml/public/jobs/new_job_new/common/job_creator/util/constants.ts` be used here. @jgowdyelastic ? Or the `CREATED_BY_LABEL` enum in that file could be moved to a more generic constants file perhaps?

Reply:
I was thinking about `CREATED_BY_LABEL`, but this enum might be extended I guess because created_by might me module values as well